app/inbound/save-settlement.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../data')
+const getPayementRequestbyInvoiceNumber = require('./get-payment-request-by-invoice-number')
+
+const saveSettlement = async (settlement, transaction) => {
+  const matchedPaymentRequest = await getPayementRequestbyInvoiceNumber(settlement, transaction)
```

Comment:
here, you can pass `settlement.invoiceNumber` and then change the signature of `getPayementRequestbyInvoiceNumber` from `(settlement, transaction)` to `(invoiceNumber, transaction)`